pkg/api/api.go

```diff
@@ -90,10 +90,10 @@ func Initialize(db *storage.Database, msgProducer message.TopicProducer, cfg *co
 			})
 			r.Route("/groups", func(r chi.Router) {
 				r.Post("/", s.Rest.CreateGroup())
-				r.Put("/enable", s.Rest.SetGroupEnabled(true))
-				r.Put("/disable", s.Rest.SetGroupEnabled(false))
 				r.Route("/{groupID}", func(r chi.Router) {
 					r.Get("/", s.Rest.GetGroupByID())
+					r.Put("/enable", s.Rest.SetGroupEnabled(true))
```

Comment:
The team talked about this on our Friday call. We think the better solution is is to do a PATCH request to the /{groupID} with the enabled field changed. After thinking about it, that seemed like the more REST-y solution.

Reply:
Switched to PATCH `/groups/{groupID}`, please take a look at implementation